Developer-guide/Building-GlusterFS.md

```diff
@@ -28,10 +28,10 @@ The following packages are required for building GlusterFS,
 
 ### Fedora
 
-The following yum command installs all the build requirements for
+The following dnf command installs all the build requirements for
```

Comment:
You need to mention the version of fedora from which `dnf` is supported Or mention `yum/dnf`